i18n/zh-cn/docusaurus-plugin-content-docs/current/design/preemption.md

```diff
@@ -0,0 +1,455 @@
+---
+id: preemption
+title: 抢佔
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# 抢佔式设计
+
+## 简介
+
+在过去很长一段时间以来，YuniKorn 的部份代码中都包含了抢佔代码，这个设计是为了去替换掉这些内容。
+
+原始代码并没有经过充分测试。
+在 [YUNIKORN-317](https://issues.apache.org/jira/browse/YUNIKORN-317) 移除核心的快取后，抢佔代码已从调度路径中移除。
+唯一的改动是抢佔代码有经过编译修正。
+在新版本抢佔设计的准备过程中，旧代码已经在 [YUNIKORN-1269](https://issues.apache.org/jira/browse/YUNIKORN-1269) 被删除。
+
+为了帮助关键的工作负载顺利运行，抢占几乎是所有调度程序都会使用的功能。
+抢佔的重要性来自系统没有这个功能就无法顺利运行的事实，例如 k8s 上的 DaemonSets，以及其他像 SLA 或优先级等因素。
+DaemonSet pod 的抢占已作为 [yunikorn-1085](https://issues.apache.org/jira/browse/YUNIKORN-1085) 的一部份实现，因此，本设计着重在其他工作负载的抢占。
+
+## 目标
+
+* 重用现有的 Kubernetes 对象
+* 实现队列间抢占
+* 使用核心调度器中的优先级资讯支援抢佔功能
+
+## 非目标
+
+* 队列内抢占
+* 跨节点抢占
+* 支援所有的优先级
+
+## 抢佔与优先级
+
+### Kubernetes
+
+整个集群中，所有命名空间内的所有 Pod 都会根据优先级进行排序。
+相较于优先级低的 Pod，优先级高的 Pod 会先被进行调度。
+Kubernetes 调度程序中的抢占也仅基于这种优先级。
+完整的文档可以在[这里](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/)找到。
```

Comment:
This might be causing the build failure. Please add spaces before and after the URL. Do not let the text line up with the start and the end of the URL like this:
```
完整的文档可以在 [这里](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/) 找到。
```